erpc_c/transports/erpc_tcp_transport.cpp

```diff
@@ -14,11 +14,9 @@
 #include <err.h>
```

Comment:
We may also consider creating a new file called `erpc_tcp_transport_zephyr.cpp` with these changes rather than modifying inplace if we ever want to use this code outside of a zephyr context.

Reply:
Actually I think we have to. Otherwise the client will be limited to Zephyr (which is exactly where we don't need it, at least for now).  Obviously we need the client as portable as possible. 
I think we  need to be able to build both the client and the server from the same tree otherwise they can grow out of sync.